app/javascript/components/widget/components/widget-composed-chart/component.jsx

```diff
@@ -14,7 +14,7 @@ class WidgetComposedChart extends Component {
     data: PropTypes.array,
     config: PropTypes.object,
     settings: PropTypes.object,
-    settingsConfig: PropTypes.object,
+    settingsConfig: PropTypes.array,
```

Comment:
Is this always true? Or was this a fix for an error 😅 ?

Reply:
I believe it should always be an array, it's an array on every other place except the widget-numbered-list/component.jsx. I think I fixed this in another branch but I'm not 100% sure, so I did it here as well to remove the error.